guide/guide-styles.css

```
@@ -0,0 +1,553 @@
+.guide-content li::marker {
+  color: #8CC63F;
+}
+.guide-content hr {
+  border: none;
+  border-top: 1px solid rgba(140, 198, 63, 0.15);
+  margin: 48px 0;
+}
+
+/* ===== CODE BLOCKS ===== */
+.guide-content code {
+  font-family: 'JetBrains Mono', 'Fira Code', 'Cascadia Code', 'Consolas', monospace;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)